classes/class-wc-connect-nux.php

```diff
@@ -186,11 +217,14 @@ public function set_up_nux_notices() {
 			}
 
 			$jetpack_install_status = $this->get_jetpack_install_status();
+			$banner_to_display = $this->get_banner_type_to_display( array(
```

Comment:
I'd prefer not to merge them since they cover two different things:
1. The Jetpack install status
2. The banner to display based on 3 factors: the JP install status, whether we have displayed the before banner, and whether TOS was accepted.

Doing it the way it's done right splits it up by function, and also allows for neater unit and integration testing.

Happy to rename the constants, though.